bin/default/alignak.in

```diff
@@ -64,8 +64,12 @@ LIB=$LIB$
 ARBITERCFG="$ETC/daemons/arbiterd.ini"
 
 # location of the alignak configuration file
-# Please update $ETC$ instead of this one.
-ALIGNAKCFG="$ETC/alignak.cfg"
+# Now look if some required variables are pre defined:
+if ! test "$ALIGNAKCFG"
```

Comment:
You are right, there are some inconsistency in the alignak script too as it uses both noatation